vsintegration/tests/unittests/Tests.LanguageService.Colorizer.fs

```diff
@@ -1,1091 +0,0 @@
-// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@dsyme  The separation is because the tests are divided into files based on functionality. In Roslyn, this division will change because different services/exporters will be responsible for different parts of functionality, and some tests will be deleted as Roslyn will take care of its functionality, so it doesn't look like a priority for me to keep the old structure.
Another example is intellisense errors. There are 'squiggle' tests scattered across 'Squiggles' and 'Script'. After we implement the diagnostics analyzer in roslyn, these will be combined into 'Diagnostics Analyzer' tests which we test that we generate the correct start/end spans and messages for incoming errors, but ignore all other VS functionality.
Do you still think we should keep the old structure?